keyboards/clueboard/60/config.h

```diff
@@ -53,10 +53,18 @@
 #define PREVENT_STUCK_MODIFIERS
 
 /* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+
```

Comment:
Could you remove the extra commented out stuff here? 

I believe you can do this by running "git checkout 53ff8a31b61952d9675558149d927f7942071df9 keyboards/clueboard/60/config.h"

Reply:
Yeah, that was already included in there...

Reply:
It may have been, but a couple of the lines got duplicated in your repo for some reason: https://github.com/lbibass/qmk_firmware/blob/f7a4540e907a71d5c87a428f705f8821c409004d/keyboards/clueboard/60/config.h#L57-L66

The above command should fix that issue.